index.js

```diff
@@ -0,0 +1,2 @@
+require('./source/js/init.js');
+module.exports = 'ngImgCrop';
```

Comment:
@CrackerakiUA Not sure I understood your question. I was able to bundle using browserify after adding this, if that's what you're asking ? I picked this up from an old issue where someone had suggested this change but did not create a PR